.travis.yml

```diff
@@ -4,13 +4,14 @@ python:
   - 2.6
   - 2.7
   - 3.5
+cache: pip
 matrix:
   allow_failures:
     - python: 3.5
 install:
+  - pip install -U pip coveralls wheel
   - pip install https://github.com/thijstriemstra/python-messaging/archive/master.zip
-  - pip install coveralls
-  - pip install wheel
+  - pip install -r requirements.txt
```

Comment:
Unfortunately it isn't covered (at least on python 3.5). [This](https://travis-ci.org/smn/txgsm/jobs/90106076) is before that change, where you see Twisted isn't found at all:

```
ImportError: No module named 'twisted.internet'
```
And [this](https://travis-ci.org/smn/txgsm/jobs/90106815) is the (expected) result after the change:

```
ImportError: No module named 'twisted.internet.serialport'
```